docs/whatsnew/7.0.rst

```diff
@@ -84,18 +84,35 @@ The minimum required versions of the following packages have been updated:
 - pandas>=2.1.0
 - aiobotocore>=2.6.0
 
-Maps now plot with autoalignment by default
-===========================================
+Polarimeter to Unify the Corona and Heliosphere (PUNCH) map source
+==================================================================
 
-The "autoalignment" functionality of the `~sunpy.map.Map` method :meth:`~sunpy.map.GenericMap.plot` has been significantly enhanced and is now enabled by default.
+There is now support for PUNCH level 1 products within sunpy.
```

Comment:
Are these level 1 products? The sample header they gave us has level 3 in the header.

Reply:
Woops.